spec/features/admins/admin_can_change_users_status_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+require 'helpers/admin_helper'
+
+RSpec.feature "Admin can change a user's status" do
```

Comment:
i don't see any test for making a user an admin or removing their admin status. am i missing something?